rskj-core/src/main/java/org/ethereum/sync/SyncPool.java

```diff
@@ -116,6 +118,8 @@ public void start() {
             }, WORKER_TIMEOUT, WORKER_TIMEOUT, TimeUnit.SECONDS
         );
 
+        emmitStartEvent();
```

Comment:
not sure this is a good place to emit `start` / `stop` events for the case of `syncing`. This `SyncPool` service is being started even if a node is already fully synced. I guess we should mimic the behaviour of the [eth_syncing](https://github.com/rsksmart/rskj/blob/master/rskj-core/src/main/java/org/ethereum/rpc/Web3Impl.java#L276) method

I believe a good candidate where this can be done is the `SyncProcessor` class. E.g. emitting `start` event in the [startDownloadingBodies](https://github.com/rsksmart/rskj/blob/master/rskj-core/src/main/java/co/rsk/net/SyncProcessor.java#L225) and `stop` - in [stopSyncing](https://github.com/rsksmart/rskj/blob/master/rskj-core/src/main/java/co/rsk/net/SyncProcessor.java#L307). What do you think?

Reply:
Yes, totally agreed! I've moved from the events from SyncPool to SyncProcessor.